contributing/contributing.md

```diff
@@ -1,3 +1,9 @@
+---
```

Comment:
Did we intend to move this here?

Reply:
Short answer: no.
Long answer: we've discussed section reorganization in #63, which doesn't move the contributing.md file into the subdirectory. This PR is a content PR for the glossary; once we have the final organization approved, we'll be able to rebase onto this PR and then merge into master.

Reply:
I just merged #63 -- so we should now be able to rebase this (64) onto master? and this file edit should go away?

We can also discuss later whether https://docs.datacommons.org/contributing/contributing_to_datacommons.html should be so similar to https://github.com/datacommonsorg/docsite/blob/master/CONTRIBUTING.md . And the style guide might also want to be linked in https://github.com/datacommonsorg/docsite/blob/master/CONTRIBUTING.md.

Reply:
@tjann I removed this doc and cleaned up the PR so it is _just_ the glossary and nothing else. It should be ready to merge at this time.

Reply:
Thanks!